config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```
@@ -1220,3 +1920,235 @@ fields:
             default:
               region: content
               weight: 100
+
+    -   name: bchem_labfacility
+        content_type: biochem_experiment
+        label: Laboratory Facility
+        type: chado_property_type_default
+        description: Include the name or code of the laboratory. Location of the laboratory needs to be included (e.g. City, Province)
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: ENVO
+            termAccession: '01001406'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_date
+        content_type: biochem_experiment
+        label: Experiment Date
+        type: chado_property_type_default
+        description: The date and time the experiment was performed (e.g. 2024Aug5 3:15pm)
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: NCIT
+            termAccession: 'C29848'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_technique
+        content_type: biochem_technique
+        label: Biochemical Technique
+        type: chado_property_type_default
+        description: The biochemical technique used to perform a particular measurement or solve a biological problem (i.e. Alanine is determined by Near Infrared Reflectance spectroscopy measurement calibrated against the regular acid hydrolysis official method).
```

The example to me sounds like an experiment name or part of the methodology. The following suggestion is another Carolyn/Lacey hybrid:

```suggestion
        description: The biochemical technique used to perform a particular measurement or solve a biological problem. If it is commonly referred to as an acronym, write out the full name followed by its acronym. For example, "Near Infrared Reflectance (NIR) spectroscopy", "Polymerase Chain Reaction (PCR)", "Optical Coherence Tomography (OCT)"
```